### PR TITLE
Dynamic section

### PR DESCRIPTION
**Pull request by zieeco**
# HTML & CSS - CAPSTONE PROJECT

**In this milestone, I did the following;** 

- Added the home and about page 
- Styled the home and about page to be responsive to a desktop and mobile device using the mobile-first approach
- Added some basics functionality with javascript to aid interactivity for UX
- Made the webpage accessible to enhance accessibility 
- Tried fixing the color contrast but was unable to fix all of it
- Ensured there was no linters error

This is the complete original version of my capstone project

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,67 @@
...
+
+👤 **Isaac Samuel**
+
+- Github: [@zieeco](https://github.com/zieeco)
+
+- LinkedIn: [Isaac Imaobong Samuel](https://www.linkedin.com/in/isaac-imaobong-samuel-a4849b1b8/)
+
+## Credits
+
+- The original design ideal by [Cindy Shin in Behance](https://www.behance.net/adagio07) from [Microverse](https://bit.ly/MicroverseTN)
```
Comment (AshakaE) [inline README.md]: Cindy shin is not from Microverse 
- [ ] Kindly change this line to " The [Original design](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015) by [Cindy Shin](https://www.behance.net/adagio07) . " You can acknowledge [Microverse](https://www.mircoverse.org) on a different line 🙂

File: about.html [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+        <li><button class="color-white-text">English</button></li>
+      </ul>
+    </nav>
+
+    <button type="button" class="toggle-btn color-dark-text">
+      <img class="bars-fw1 bars-fw1-fs" src="./images/iconmonstr-menu-1.svg" alt="harmburger" aria-hidden="true">
+    </button>
+
+    <nav class="lg-navbar bg-white">
+      <a class="namm-logoo-a" href="./"><img class="namm-logoo" src="./images/download.png" alt="namm logo"></a>
```
Comment (AshakaE) [inline about.html]: For all places where this logo is used

- [ ] Kindly change the color to follow the color scheme of the original design 🙏

File: about.html [hunk 3]
```diff
@@ -0,0 +1,165 @@
...
+        Partners
+      </h2>
+      <div class="line bg-primary partners-line"></div>
+
+      <div class="partners-logos">
+        <img class="partners-image image-container" src="./images/partners/image2vector.svg" alt="mozilla ">
+        <img class="partners-image image-container" src="./images/partners/google.svg" alt="google">
+        <img class="partners-image image-container" src="./images/partners/naver.svg" alt="naver">
+        <img class="partners-image image-container" src="./images/partners/tokopedia.png" alt="zoom">
+        <img class="partners-image image-container" src="./images/airbnb.svg" alt="tokopedia">
```
Comment (AshakaE) [inline about.html:150]: For all places with `partners-logo`

- [ ] Kindly keep the color scheme this section to one, from the design all logos here should be in grayscale 🙄

File: index.html [hunk 4]
```diff
@@ -0,0 +1,172 @@
...
+            global
+            communities, continue to power our industry's recovery, and create a more musical world.</p>
+        </div>
+        <div class="control-header-1">
+          <h3 class="header-2 color-dark-text ff-2">2022.20.01(THUR) ~ 23 (SUN)</h3>
+        </div>
+        <p class="color-dark-text">Anaheim Convention Center • Southern California <br><span
+            class="time color-dark-text ff-2"> - 10-AM PROMPT</span></p>
+      </div>
+    </section>
```
Comment (AshakaE) [inline index.html:76]: This section is having too much white space

- [ ] Kindly reduce the space here
_See reference_
image
image

**Review by AshakaE — CHANGES_REQUESTED** [review]
### Hi @zieeco  🙋‍♀️, 

Good job so far! 👌
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


#  Required Changes ♻️ 🚧
_Check the comments under the review._

- [ ] Kindly increase the font size for the webpages 🙂 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by george-shammar — APPROVED** [review]
Hi @zieeco , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

:heavy_check_mark: You've done a pretty good job with the design

Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**